lib/aws-manager.js

```diff
@@ -1037,12 +1037,20 @@ class AwsManager {
     assert(this.__availableAZ[bid.region].includes(bid.zone),
         'will not submit spot request in an unavailable az');
 
-    let spotRequest = await this.ec2.requestSpotInstances.inRegion(bid.region, {
-      InstanceCount: 1,
-      Type: 'one-time',
-      LaunchSpecification: launchInfo.launchSpec,
-      SpotPrice: bid.price.toString(),
-    });
+    try {
+      let spotRequest = await this.ec2.requestSpotInstances.inRegion(bid.region, {
+        InstanceCount: 1,
+        Type: 'one-time',
+        LaunchSpecification: launchInfo.launchSpec,
+        SpotPrice: bid.price.toString(),
+      });
+    } catch (err) {
+      if (err.code === 'InvalidAMIID.NotFound') {
+        debug(err.message + 'in ' + bid.region);
```

Comment:
probably using `console.log` I'm starting to think we should just use that for things that aren't debug messages...